packages/backend-storage/API.md

```diff
@@ -20,7 +25,30 @@ export type AmplifyStorageProps = {
 };
 
 // @public
-export const defineStorage: (props: AmplifyStorageProps) => ConstructFactory<ResourceProvider<StorageResources>>;
+export const defineStorage: (props: AmplifyStorageFactoryProps) => ConstructFactory<ResourceProvider<StorageResources>>;
+
+// @public (undocumented)
+export type EntityAccessBuilder = {

```

Comment:
what is `Entity` referring to here ?

Reply:
It's any "runtime entity" in the backend, eg auth role, unauth role, lambda execution role. I can probably just change this to `RoleAccessBuilder`

Reply:
`RoleAccessBuilder` sounds good.
Or maybe `IAMEntityAccessBuilder` (in case we'll ever support something else than role like iam user group).

Reply:
I'll go with RoleAccessBuilder for now because if we ever want to support something besides roles that will likely require a larger refactor effort